docs/governance/minutes/2019-05-15.rst

```diff
@@ -0,0 +1,284 @@
+.. -*- mode: rst -*-
+
+May 15, 2019
+============
```

Comment:
The broad structure of this document is:

1. Agenda
2. Votes and Resolutions
3. Semester and Roadmap
4. Minutes
5. Action Items 

I figured this would be the most general way to contain information for all minutes. Votes and resolutions are usually the most important thing people want to look at, which is why it's at the top. Semester and Roadmap is the core of the semester meetings, so it has its own section. The minutes contain the other discussion items in their own headers (e.g. PyCon report, GSoC, etc.) However, because of this, the Agenda is the only thing that shows the order of discussion. Also, it isn't easy to find things because the ToC on the sidebar doesn't go to that depth.